appveyor.yml

```diff
@@ -1,132 +1,135 @@
-- echo XERCES_LIB=$(SDK_LIB)\xerces-c_3.lib >> nmake.local
-- echo ILI_ENABLED = YES >> nmake.local
-- echo EXPAT_DIR=bla >> nmake.local
-- echo EXPAT_INCLUDE=-I$(SDK_INC) >> nmake.local
-- echo EXPAT_LIB=$(SDK_LIB)\expat.lib >> nmake.local
-- echo LIBKML_DIR=bla >> nmake.local
-- echo LIBKML_INCLUDE=-I$(SDK_INC) >> nmake.local
-- echo LIBKML_LIBS=$(SDK_LIB)\libkmlbase.lib $(SDK_LIB)\libkmlconvenience.lib $(SDK_LIB)\libkmldom.lib $(SDK_LIB)\libkmlengine.lib $(SDK_LIB)\libkmlregionator.lib $(SDK_LIB)\libkmlxsd.lib $(SDK_LIB)\expat.lib $(SDK_LIB)\zdll.lib >> nmake.local
-- echo POPPLER_ENABLED = YES >> nmake.local
-- echo POPPLER_CFLAGS = -I$(SDK_INC) -I$(SDK_INC)\poppler >> nmake.local
-- echo POPPLER_HAS_OPTCONTENT = YES >> nmake.local
-- echo POPPLER_0_20_OR_LATER = YES >> nmake.local
-- echo POPPLER_0_23_OR_LATER = YES >> nmake.local
-- echo POPPLER_BASE_STREAM_HAS_TWO_ARGS = YES >> nmake.local
-- echo POPPLER_LIBS = $(SDK_LIB)\poppler.lib $(SDK_LIB)\freetype2411.lib advapi32.lib gdi32.lib >> nmake.local
-- echo OPENJPEG_ENABLED = YES >> nmake.local
-- echo OPENJPEG_CFLAGS = -I$(SDK_INC)\openjpeg-2.1 >> nmake.local
-- echo OPENJPEG_LIB = $(SDK_LIB)\openjp2.lib >> nmake.local
-- echo ECWDIR=bla >> nmake.local
-- echo ECWLIB=$(SDK_LIB)\libecwj2.lib >> nmake.local
-- echo ECWFLAGS=-DECWSDK_VERSION=33 -I$(SDK_INC) /D_MBCS /D_UNICODE /DUNICODE /D_WINDOWS  /DLIBECWJ2 /DWIN32 /D_WINDLL -DNO_X86_MMI >> nmake.local
-#- echo LIBICONV_DIR = bla >> nmake.local
-#- echo LIBICONV_INCLUDE = -I$(SDK_INC) >> nmake.local
-#- echo LIBICONV_LIBRARY = $(SDK_LIB)\iconv.lib >> nmake.local
-#- echo LIBICONV_CFLAGS = -DICONV_CONST=const >> nmake.local
-#- echo HDF4_DIR = $(SDK_PREFIX) >> nmake.local
-#- echo HDF4_LIB = $(SDK_LIB)\hdfdll.lib $(SDK_LIB)\mfhdfdll.lib Ws2_32.lib >> nmake.local
-- echo HDF5_DIR = $(SDK_PREFIX) >> nmake.local
-- echo HDF5_LIB = $(SDK_LIB)\hdf5.lib  >> nmake.local
-- echo KEA_CFLAGS = -I$(SDK_INC) >> nmake.local
-- echo KEA_LIB = $(SDK_LIB)\libkea.lib >> nmake.local
-- echo NETCDF_SETTING=yes >> nmake.local
-- echo NETCDF_LIB=$(SDK_LIB)\netcdf.lib >> nmake.local
-- echo NETCDF_INC_DIR=$(SDK_INC) >> nmake.local
-- echo NETCDF_HAS_NC4 = yes >> nmake.local
-- nmake /f makefile.vc MSVC_VER=%MSVC_VER% %WIN64_ARG% USER_DEFS=/WX
-- cd apps
-- nmake /f makefile.vc test_ogrsf.exe MSVC_VER=%MSVC_VER% %WIN64_ARG% USER_DEFS=/WX
-- cd ..
-- SET DISTUTILS_USE_SDK=1
-- SET MSSdk=1
-- cd swig\python
-- python setup.py build
-- python setup.py install
-- cd ..\..
-- cd ..\autotest\cpp
-- nmake /f makefile.vc MSVC_VER=%MSVC_VER% %WIN64_ARG%
-- cd ..\..
+init:
+  - ps: |
+      function exec
+      {
+        param ( [ScriptBlock] $ScriptBlock )
+        & $ScriptBlock 2>&1 | ForEach-Object -Process { "$_" }
+        if ($LastExitCode -ne 0) { exit $LastExitCode }
+      }
+  - ps: iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-rdp.ps1'))
```

Comment:
Sorry to be late with my comments, but it may be worth noting that this line, if not commented, will make AppBeyor to keep the build up and waiting for 60 minutes, even if successful. I'm not 100% sure, but this will block any upcoming builds for 60 mins.

Reply:
To be honest I can't make any sense of this powershell syntax, I just checked that the AppVeyor builds were happy.
Are you sure about this 60 minute blocking ? From https://ci.appveyor.com/project/OSGeo/gdal/build/1.0.4937, it looks like the builds completed in a reasonable time.

Reply:
@rouault I wasn't sure and my suspicious were not valid, turns out.

The [build is blocked](https://www.appveyor.com/docs/how-to/rdp-to-build-worker/) but IFF `$blockRdp = $true;` is also specified, that is not the case here. The change looks good.

Reply:
If you want to block and investigate in 60min limit, syntax is as follows:
```
on_failure:
  - ps: $blockRdp = $true; iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-rdp.ps1'))
```
Current syntax is to allow to enter VM during build. It is why I put it in init section.

It may be worth writing both syntax and commented it out  for developer to enable it when necessary.

Reply:
Agreed that some documentation would be good

Reply:
Here is an official documentation https://www.appveyor.com/docs/how-to/rdp-to-build-worker/
I'd like to write some comment pointing here in appveyor.yml